### PR TITLE
[docs] Add charts to MUI X introduction pages

### PR DESCRIPTION
**Pull request by joserodolfofreitas**
## Summary

- Mention charts on the overview
- Add chart installation instructions

## Preview

https://deploy-preview-9704--material-ui-x.netlify.app/x/introduction/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,11 @@
 
 <h1 align="center">MUI X</h1>
 
-MUI X is a collection of advanced UI components, including a [data grid](https://mui.com/x/react-data-grid/) and [date and time pickers](https://mui.com/x/react-date-pickers/getting-started/).
+MUI X is a collection of advanced UI components, including:
+
+- [Data Grid](https://mui.com/x/react-data-grid/)
```
Comment (joserodolfofreitas) [inline README.md:10]: Maybe makes sense adding imgs for the components

Comment (alexfauquette) [reply README.md:10]: Make sense, but the effort of introducing a component in the markdown page to have nice-looking images might does not worth the benefit.

For me it's already an improvement as it is 👍

**Review by richbustos — APPROVED** [review]
Very nice!

File: README.md (hunk 1 above)
Comment (joserodolfofreitas) [reply README.md]: I realized the list was duplicated, so I removed it altogether.

**Pull request merged**